docs/quickstart.md

```diff
@@ -189,4 +192,6 @@ Now we can print the best coefficient.
 {'regularizer__regularizer_strength': 0.001}
 ```
 
+For a more detailed example you can checkout our [tutorial on cross-validation](/generated/api_guide/plot_06_sklearn_pipeline_cv_demo).
```

Comment:
this path is incorrect, and I don't think we need the link appearing twice back-to-back. My inclination would be to merge the two and place it here:
```suggestion
For more information, including a practical example on how to construct a parameter grid and cross-validate hyperparameters across an entire pipeline, please refer to the [tutorial on pipelining and cross-validation](../generated/api_guide/plot_06_sklearn_pipeline_cv_demo).
```